### PR TITLE
Update format.md

### PR DESCRIPTION
**Pull request by mr-tz**
Per discussion in #266.

### CODE REVIEW

File: doc/format.md [hunk 1]
```diff
@@ -145,13 +145,18 @@ It can be referenced in other rules, so if you change a rule name, be sure to se
 By convention, the rule name should complete one of the following sentences:
   - "The program/function may..."
   - "The program was..."
-  
+
+To focus rule names we try to omit articles (the/a/an).
```
Comment (mr-tz) [inline doc/format.md:149]: My preference, but happy to debate this.

Comment (williballenthin) [reply doc/format.md:149]:
```suggestion
To focus rule names we try to omit articles (the/a/an).
For example, prefer `make HTTP request` over `make an HTTP request`.
```

File: doc/format.md [hunk 2]
```diff
@@ -145,13 +145,18 @@ It can be referenced in other rules, so if you change a rule name, be sure to se
 By convention, the rule name should complete one of the following sentences:
   - "The program/function may..."
   - "The program was..."
-  
+
+To focus rule names we try to omit articles (the/a/an).
+
 When the rule describes a specific means to implement a techinque, this is typically specified by "via XYZ".
```
Comment (williballenthin) [inline doc/format.md:151]:
```suggestion
When the rule describes a specific means to implement a technique, this is typically specified by "via XYZ".
For example, `make HTTP request via WinInet` or `make HTTP request via libcurl`.
```

**Review by williballenthin — APPROVED** [review]
nice!

some suggestions, but only suggestions, i dont feel strongly that they be incorporated.

**Comment by mr-tz** [comment]
Good improvements! Thanks.

**Comment by mr-tz** [comment]
Lint fail see #283

**Pull request merged**